lib/polyfill/patchedmediakeys_safari.js

```diff
@@ -653,118 +644,6 @@ shaka.polyfill.PatchedMediaKeysSafari.MediaKeySession.prototype.
- *
- * @param {string} initDataType
- * @param {BufferSource} initData
- * @return {BufferSource}
- * @throws {Error} if we do not have a server certificate
- * @private
- */
-shaka.polyfill.PatchedMediaKeysSafari.MediaKeySession.prototype.wrapInitData_ =
-    function(initDataType, initData) {
-  if (!this.isFairPlay_()) {
-    // Don't touch non-FairPlay init data.
-    shaka.log.debug('Not FairPlay.  Using init data as-is.');
-    return initData;
-  }
-
-  shaka.log.debug('Wrapping FairPlay init data.');
-  var serverCertificate = this.mediaKeys_.serverCertificate;
-  if (!serverCertificate) {
-    throw new Error('A server certificate is required for FairPlay!');
-  }
-
-  // skd is the type used by the HLS parser when it feeds us an skd URI as init
-  // data.  This is how we get the content ID.
-  if (initDataType != 'skd') {
-    throw new Error('Only skd-type init data is supported for FairPlay!');
-  }
-
-  // TODO: is this the right approach to parse the string?
-  // Step 1: extract the content ID from the init data, which is a little-endian
-  // UTF-16 string.
-  var initDataAsString = shaka.util.StringUtils.fromUTF16(
-      initData, /* littleEndian */ true);
-  shaka.log.info('FairPlay init data:', initDataAsString);
-  var initDataAsUri = new goog.Uri(initDataAsString);
-  var contentIdAsString = initDataAsUri.getDomain();
-  shaka.log.info('FairPlay content ID:', contentIdAsString);
-  var contentIdAsArray = shaka.util.StringUtils.toUTF16(
-      contentIdAsString, /* littleEndian */ true);
-  // Store this for later.  We may need it in the license request.
-  this.contentId_ = contentIdAsString;
-
-  // Step 2: construct the individual pieces of the formatted init data.
-  // Format: [original init data] + [LE 4 byte content ID byte length] +
-  //         [UTF-16-LE content ID] + [LE 4 byte certificate byte length] +
-  //         [certificate bytes]
-  var idLength = new Uint8Array(4);
-  (new DataView(idLength.buffer)).setUint32(
-      0, contentIdAsArray.byteLength, /* littleEndian */ true);
-  var certificateLength = new Uint8Array(4);
-  (new DataView(certificateLength.buffer)).setUint32(
-      0, serverCertificate.byteLength, /* littleEndian */ true);
-
-  // Step 3: concatenate the pieces and return an ArrayBuffer.
-  // TODO: examples show original init data first, but WebKit's sources seem
-  // to show parsers that expect not to have that piece.
-  var newInitData = shaka.util.Uint8ArrayUtils.concat(
-      new Uint8Array(initData),
-      idLength,
-      new Uint8Array(contentIdAsArray),
-      certificateLength,
-      new Uint8Array(serverCertificate));
-  return newInitData.buffer;
-};
-
-
-/**
- * Unwrap the license response into the format the CDM expects.
- *
- * @param {BufferSource} response
- * @return {BufferSource}
- * @throws {Error} if we fail to parse the reponse
- * @private
- */
-shaka.polyfill.PatchedMediaKeysSafari.MediaKeySession.prototype.unwrapLicense_ =
```

Comment:
Here as well, I've never seen CKC-wrapped license responses from the servers I've worked with. The application can handle this via response filters.